### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by k0da**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  golangci:
+    runs-on: ubuntu-latest
+    name: lint
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+      - name: golang-ci
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
```
Comment (kuritka) [inline .github/workflows/test.yaml]:
```suggestion
          version: v1.32
```

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          args: --deadline 10m
```
Comment (kuritka) [inline .github/workflows/test.yaml]: up to you, consider < 2m

**Review by kuritka — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yaml (hunk 1 above)
Comment (k0da) [reply .github/workflows/test.yaml]: Done

File: .github/workflows/test.yaml (hunk 2 above)
Comment (k0da) [reply .github/workflows/test.yaml]: Done

**Review by kuritka — APPROVED** [review]

**Pull request merged**